### DISCUSSION TITLE
Error in buildings['adjacency'] calculation

### DISCUSSION THREAD

**Discussion by cdkang** [pysal/momepy#431] [Q&A] [answered]
Hello, 

I appreciate your comments on my questions. 

Please let me know how to delete this discussion as I have solved my trouble. 

Sincerely,

**Comment by jGaboardi** [comment, marked as answer]
@cdkang From the traceback this appears to be a problem within `scipy.spatial`. In order to help us help you more efficiently, please copy and paste the full traceback you are getting (for this error and future ones). Screen shots are difficult to work with tracking down solutions on StackOverflow! 😆

[Here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code) is a guide for code formatting in GitHub Markdown.

**Comment by jGaboardi** [comment]
@cdkang No need to delete the discussion or the traceback! In fact, please leave it up since it might help others in the future!
